auth.md

```diff
@@ -0,0 +1,41 @@
+# Auth
+
+By default, we're using cookies as a mock auth method. It's for trying out OpenGPTs.
+For production, we recommend using JWT auth, outlined below.
+
+## JWT Auth: Options
+
+There are two ways to use JWT: Local and OIDC. The main difference is in how the key
+used to decode the JWT is obtained. For the Local method, you'll provide the decode
+key as a Base64-encoded string in an environment variable. For the OIDC method, the
+key is obtained from the OIDC provider automatically.
+
+### JWT OIDC
+
+If you're looking to integrate with an identity provider, OIDC is the way to go.
+It will figure out the decode key for you, so you don't have to worry about it.
+Just set `AUTH_TYPE=jwt_oidc` along with the issuer and audience. Audience can
+be one or many - just separate them with commas.
+
+```bash
+export AUTH_TYPE=jwt_oidc
+export JWT_ISS=<issuer>
```

Comment:
I don't see where these environment variables are read. Can you clarify how they're used?

Reply:
They are read in `backend/app/auth/settings.py` via pydantic's `BaseSettings`. Pydantic takes care of extracting the env vars.

Reply:
Got it!

Reference: https://docs.pydantic.dev/latest/concepts/pydantic_settings/#usage